### PR TITLE
Fix/issue 36

### PR DESCRIPTION
**Pull request by mbehr1**

### CODE REVIEW

**Comment by mbehr1** [comment]
@christian-bromann see here a set of commits that fix issue #36.
There is one change (locators) I'm not really sure about. It could as well be introduced in a later version or a different kind of solution might be better. Please have a look.
There are other getters that are not working with those items as well (e.g. title returns often null for some elements). I added a few todos. Here more tests are needed.

File: src/pageobjects/sidebar/ViewContent.ts [hunk 1]
```diff
@@ -50,7 +50,8 @@ export class ViewContent extends BasePage<typeof ViewContentLocators> {
 
         const availableSections: Set<string> = new Set()
         for (const element of elements) {
-            const sectionTitle = await this.sectionTitle$.getText()
+            const sectionTitle = await (await element.$(this.locators.sectionTitle)).getText()
```
Comment (christian-bromann) [inline src/pageobjects/sidebar/ViewContent.ts:53]: We can chain the promise calls here:
```suggestion
            const sectionTitle = await element.$(this.locators.sectionTitle).getText()
```

File: src/pageobjects/sidebar/ViewContent.ts [hunk 2]
```diff
@@ -50,7 +50,8 @@ export class ViewContent extends BasePage<typeof ViewContentLocators> {
...
+            // todo .getAttribute('title') imho would be better e.g. 'mount' instead of '/ [TEST FILES]'
```
Comment (christian-bromann) [inline src/pageobjects/sidebar/ViewContent.ts:54]: Is this still pending? I rather would like to create an issue for this than keeping it as a todo in the code.

**Review by christian-bromann — COMMENTED** [review]
This is great, thanks! Some comments

File: src/pageobjects/sidebar/ViewContent.ts (hunk 1 above)
Comment (mbehr1) [reply src/pageobjects/sidebar/ViewContent.ts:53]: sure. thx. shall I just use the "Commit suggestion" feature or rewrite the commit?

File: src/pageobjects/sidebar/ViewContent.ts (hunk 2 above)
Comment (mbehr1) [reply src/pageobjects/sidebar/ViewContent.ts:54]: having an issue is fine. I can create one. Shall I rewrite this commit or add a commit on top that removes it?

Comment (christian-bromann) [reply src/pageobjects/sidebar/ViewContent.ts:54]: > Shall I rewrite this commit or add a commit on top that removes it?

Just add one on top as we gonna squash the PR anyway.

File: src/pageobjects/sidebar/ViewContent.ts (hunk 1 above)
Comment (mbehr1) [reply src/pageobjects/sidebar/ViewContent.ts]: ok. done

File: src/pageobjects/sidebar/ViewContent.ts (hunk 2 above)
Comment (mbehr1) [reply src/pageobjects/sidebar/ViewContent.ts]: done

**Comment by mbehr1** [comment]
hmm. the failed test seems to be flaky.
It's the check in 
it('should have no webviews in the beginning', async () => {
            const workbench = await browser.getWorkbench()
            expect(await workbench.getAllWebviews()).toHaveLength(0)
        })

that finds a web view at start...
(unrelated to my changes, I guess?)

[chrome 104.0.5112.102 windows #0-0] Received array:  [{"_baseElem": [Element], "_locators": [Object], "_parentElem": undefined, "locatorKey": "WebView"}]
[6118](https://github.com/webdriverio-community/wdio-vscode-service/runs/8194156892?check_suite_focus=true#step:6:6119)
[chrome 104.0.5112.102 windows #0-0]     at Context.<anonymous> (D:\a\wdio-vscode-service\wdio-vscode-service\test\specs\basic.e2e.ts:430:54)

**Comment by mbehr1** [comment]
I have no mean to retrigger tests. @christian-bromann can you? Anything I can do?

**Comment by christian-bromann** [comment]
rerunning...

**Review by christian-bromann — APPROVED** [review]
Thanks a lot 👍

**Pull request merged**